src/createAnimatedComponent.js

```diff
@@ -292,15 +306,18 @@ export default function createAnimatedComponent(Component) {
         <Component {...props} ref={this._setComponentRef} {...platformProps} />
       );
     }
-
-    // A third party library can use getNode()
-    // to get the node reference of the decorated component
-    getNode() {
-      return this._component;
-    }
   }
 
-  AnimatedComponent.displayName = `AnimatedComponent(${Component.displayName || Component.name || 'Component'})`
-
-  return AnimatedComponent;
+  AnimatedComponent.displayName = `AnimatedComponent(${Component.displayName ||
+    Component.name ||
+    'Component'})`;
+
+  return React.forwardRef(function AnimatedComponentWrapper(props, ref) {
+    return (
+      <AnimatedComponent
+        {...props}
+        {...(ref == null ? null : { forwardedRef: ref })}
```

Comment:
If refs are only `null | T` then I think we should use `===` here

Reply:
Are they enforced to be null | T in pure JS without type checking? If not I'd leave it as is, cause I found that it was this way in Facebook's RN repo. Also type coercion table tells me that it will resolve to true only in case undefined is passed. I think it might be safe. On the other hand I'm thinking if we shouldn't be more explicit. Relying on JS type coercion in equality check is something that a lot of people may interpret wrong in the future. What do you think?

Reply:
Yeah, it'll catch `undefined | null` there, same goes with `L#232`

Doesn't hurt to leave it as it is, so I'm cool with the current solution